packages/resource-deployment/scripts/run-command-when-batch-nodes-are-idle.sh

```diff
@@ -7,17 +7,18 @@ set -eo pipefail
 
 exitWithUsageInfo() {
     echo "
-Usage: $0 -r <resource group> -c <command to run> -n <command name>
+Usage: $0 -r <resource group> -c <command to run> -n <command name> [-p <pool id (if not specified, will wait for all pools to become idle)>]
```

Comment:
pool id or pool is list?

Reply:
I just changed it to accept an array of pools